mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+SRC_URI="https://github.com/pazz/alot/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+DEPEND="
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/mock[${PYTHON_USEDEP}] )
+	"
+RDEPEND="
+	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
+	dev-python/python-magic[${PYTHON_USEDEP}]
+	>=dev-python/twisted-core-10.2.0[${PYTHON_USEDEP}]
+	>=dev-python/urwid-1.1.0[${PYTHON_USEDEP}]
+	>=dev-python/urwidtrees-1.0[${PYTHON_USEDEP}]
+	net-mail/mailbase
+	>=net-mail/notmuch-0.13[crypt,python]
+	"
+
+python_prepare_all() {
+	find "${S}" -name '*.py' -exec sed -e '1i# -*- coding: utf-8 -*-' -i '{}' \;
+
+	distutils-r1_python_prepare_all
+}
+
+python_compile_all() {
+	if use doc; then
+		emake -C docs html
+		HTML_DOCS=( docs/build/html/. )
+	fi
+}
+
+python_src_install_all() {
+	distutils-r1_src_install
+
+	dodir /usr/share/alot
+	insinto /usr/share/alot
+	doins -r extra
+}
+
+pkg_postinst() {
+	if has_version "<${CATEGORY}/${PN}-0.3.2"; then
```

Comment:
We use REPLACING_VERSIONS for it. Which is plural, since most people don't seem to get it.

Reply:
Ah I didn't know this one I changed this part to take advantage of this variable, so I added a message for when you install it for the first time, then the gpg_key to add when upgrading the 0.5.1, I think it is better.